http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -101,8 +101,13 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
         case x: CloudContext.Azure => F.pure(x.value)
       }
       hcName = RelayHybridConnectionName(params.runtime.runtimeName.asString)
-      primaryKey <- azureRelay.createRelayHybridConnection(params.relayeNamespace, hcName, cloudContext)
+      primaryKey <- azureRelay.createRelayHybridConnection(params.landingZoneResources.relayNamespace,
+                                                           hcName,
+                                                           cloudContext
+      )
       createDiskAction = createDisk(params, auth)
+      // TODO make a ticket. We're still calling WSM createVM API, which depends on a WSM owned network. We're migrating
+      // to the shared LZ subnet. We could create the new VM directly from Leo or make an updated API in WSM
```

Comment:
Hm - I had thought the `networkId` field in the WSM `createVm` payload was optional, and it would use the LZ network if not present. Is this not working? I can check too..

Reply:
The problem is that the Hybrid Connection is currently created by Leo using the old Azure Relay/network. So what needs to happen there is:
1. I need to close out the changes for the createHybridConnection API in WSM!
1. Leo can migrate to calling the API and using the LZ relay namespace.